payments-ui-core/src/test/java/com/stripe/android/ui/core/elements/CardNumberConfigTest.kt

```diff
@@ -10,7 +12,7 @@ import org.junit.Test
 import com.stripe.android.R as StripeR
 
 class CardNumberConfigTest {
-    private val cardNumberConfig = CardNumberConfig()
+    private val cardNumberConfig = CardNumberConfig(isCBCEligible = false, cardBrandFilter = DefaultCardBrandFilter)
```

Comment:
Can we also test isCbcEligible = true?

Reply:
Added! I tried to use `@ParameterizedTest` for this but it wasn't working and I don't see any usage in the project, so I took the naive approach since it's just true/false.